src/modules/edit.js

```diff
@@ -22,28 +14,15 @@ export default function edit(e) {
 
     const removeTaskFromArray = item.parentElement.previousElementSibling.children[0].getAttribute('name');
 
-    tasks = tasks
-      .filter((newTask) => {
-        if (newTask.index !== +removeTaskFromArray) {
-          return true;
-        }
-      })
-      .sort((a, b) => {
-        if (a.index !== 1) {
-          a.index = 1;
-        }
-        return a.index - b.index;
-      })
-      .sort((task, newTask) => {
-        if (task.index + 1 === newTask.index) {
-          return;
-        } else {
-          newTask.index = task.index + 1;
-        }
-      });
+    tasks = tasks.filter((newTask) => {
+      if (newTask.index !== +removeTaskFromArray) return true;
+    });
```

Comment:
- You can `return false` in the next line, it will fix the return issue. Same you can do in `add.js` as well.

Reply:
@Dipeshtwis  Thank you so much.